### PR TITLE
QA-4833 [YCSB] Handle migration to storage profiles

### PR DESCRIPTION
**Pull request by nsivkov**
https://ggsystems.atlassian.net/browse/QA-4833

Test run example: https://ward.gridgain.com/tests/6619604c1570869f793d1b10

### CODE REVIEW

File: ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java [hunk 1]
```diff
@@ -115,9 +115,17 @@ public abstract class IgniteAbstractClient extends DB {
...
   /**
    * Used to choose storage engine (e.g., 'aipersist' or 'rocksdb').
+   * @deprecated Removed in <a href="https://ggsystems.atlassian.net/browse/IGN-23905">IGN-23905</a>
    */
+  @Deprecated
   protected static String dbEngine;
 
+  /**
+   * Used to choose storage profiles separated by comma
+   * (e.g., 'default', 'default_aimem', 'default_aipersist,default_rocksdb', etc.).
```
Comment (Berkof) [inline ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java:125]:
```suggestion
   * (e.g., 'default', 'default_aimem', 'default_aipersist', 'default_rocksdb', etc.).
```

File: ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java [hunk 2]
```diff
@@ -115,9 +115,17 @@ public abstract class IgniteAbstractClient extends DB {
...
+   */
+  protected static String storageProfiles;
```
Comment (Berkof) [inline ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java:127]: Why not to use List<String> for the config parameters that is a list string by its nature?

**Review by ahitrin — APPROVED** [review]
LGTM

File: ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java (hunk 1 above)
Comment (nsivkov) [reply ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java:125]: Actually, I wanted to give an example of several storage profiles sent in one line.
As per [documentation of Zone declaration](https://docs.google.com/document/d/1VHNybikLk4I-ULJjkupuGIN2D9kxF7obRTQ7cNwBeLo/edit#bookmark=kix.h0t8uyplp4zr):
```
create zone z1 with storage_profiles='lru_rocks,clock_aipersist'; 
```

File: ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java (hunk 2 above)
Comment (nsivkov) [reply ignite3/src/main/java/site/ycsb/db/ignite3/IgniteAbstractClient.java:127]: It's just a string we grab from property and send unchanged to the expression `STORAGE_PROFILES='%s'`

**Review by Berkof — APPROVED** [review]

**Pull request merged**